examples/basics/example_intro_tutorial.py

```diff
@@ -0,0 +1,772 @@
+"""Introductory Tutorial
+======================
+
+This tutorial will illustrate the basics of how to use :code:`|toqito⟩`. This
+will cover how to instantiate and use the fundamental objects that
+:code:`|toqito⟩` provides; namely quantum states, channels, and measurements.
+"""
+
+# %% This is an introduction to the functionality in :code:`|toqito⟩` and is not meant to serve as an
+# introduction to quantum information. For introductory material on quantum
+# information, please consult "Quantum Information and Quantum Computation" by
+# Nielsen and Chuang or the freely available lecture notes `"Introduction to
+# Quantum Computing" <https://cs.uwaterloo.ca/~watrous/QC-notes/>`_
+# by John Watrous.
```

Comment:
Sorry, I know this is not a change you made, but could you change the links to the books here to use `:cite:` along with `refs.bib` keys?